packages/@aws-cdk-testing/cli-integ/tests/cli-integ-tests/cli.integtest.ts

```diff
@@ -780,6 +780,19 @@ integTest('deploy stack without resource', withDefaultFixture(async (fixture) =>
     .rejects.toThrow('conditional-resource does not exist');
 }));
 
+integTest('deploy --ignore-no-stacks', withDefaultFixture(async (fixture) => {
```

Comment:
Not sure about this.
We should verify that the app deploys when there are no stacks in it.